packages/angular-output-target/src/types.ts

```diff
@@ -1,12 +1,28 @@
 export interface OutputTargetAngular {
-  componentCorePackage?: string;
-  directivesProxyFile: string;
+  /**
+   * The package name of the component library.
+   * This is used to generate the import statements.
+   */
+  componentCorePackage: string;
+  /**
+   * @deprecated Use `proxyDeclarationFile` instead. This property has been replaced.
+   */
+  directivesProxyFile?: string; // TODO: Remove this type once the example project uses the local dep vs. remote.
   directivesArrayFile?: string;
   directivesUtilsFile?: string;
+  /**
+   * The path to the proxy file that will be generated. This can be an absolute path
+   * or a relative path from the root directory of the Stencil library.
+   */
+  proxyDeclarationFile: string;
   valueAccessorConfigs?: ValueAccessorConfig[];
   excludeComponents?: string[];
   includeImportCustomElements?: boolean;
   customElementsDir?: string;
+  /**
+   * `true` to generate a single component Angular module for each component.
+   */
+  createSingleComponentAngularModules?: boolean;
```

Comment:
Optional: The other APIs use "include" and "exclude" verbiage. Should we align with that and name this `includeSingleComponentAngularModules`?